### PR TITLE
chore: update config_change source for diff changes

### PR DESCRIPTION
**Pull request by yashmehrotra**

### CODE REVIEW

File: db/update.go [hunk 1]
```diff
@@ -838,6 +838,7 @@ func generateConfigChange(ctx api.ScrapeContext, newConf, prev models.ConfigItem
 		Diff:       &diff,
 		Patches:    string(patch),
 		Summary:    strings.Join(utils.ExtractLeafNodesAndCommonParents(patchJSON), ", "),
+		Source:     v1.ChangeTypeDiffSource,
```
Comment (moshloop) [inline db/update.go:841]:
```suggestion
		Source:     v1.ChangeTypeDiff
```

Comment (yashmehrotra) [reply db/update.go]: Updated

File: db/update.go [hunk 2]
```diff
@@ -838,6 +838,7 @@ func generateConfigChange(ctx api.ScrapeContext, newConf, prev models.ConfigItem
 		Diff:       &diff,
 		Patches:    string(patch),
 		Summary:    strings.Join(utils.ExtractLeafNodesAndCommonParents(patchJSON), ", "),
+		Source:     v1.ChangeTypeDiff,
```
Comment (adityathebe) [inline db/update.go:841]: we use "config-db" as the source for changes generated by config-db.
like health changes.
playbookStarted, ...

so let's use that probably

Comment (yashmehrotra) [reply db/update.go:841]: I intentionally didnt want that to keep the diff changes separate since their updated timestamps are handled differently

**Pull request merged**